src/fundus/publishers/us/the_intercept.py

```diff
@@ -55,3 +58,11 @@ def topics(self) -> List[str]:
                 return []
 
             return [keyword[9:] for keyword in keywords if keyword.startswith("Subject: ")]
+
+    class V1_1(V1):
+        VALID_UNTIL = date.today()
+        _summary_selector = XPath(
+            "//p[@class='post__excerpt'] | //h2[preceding-sibling::h1[contains(@class, 'post__title')]]"
+        )
+        _paragraph_selector = CSSSelector("div.entry-content > div.entry-content__content > p")
```

Comment:
In the paragraph selector, I think we should also add p blocks that are children of a blockquote block, as it is used in this article for example: https://theintercept.com/2024/02/28/congress-pakistan-election-recognition/

Reply:
I added block quotes as well